_partials/_migrate_post_schema_caggs_etal.md

```diff
@@ -0,0 +1,165 @@
+## Migrate schema post-data
+
+When you have migrated your table and hypertable data, migrate your PostgreSQL
+schema post-data. This includes information about constraints.
+
+<Procedure>
+
+### Migrating schema post-data
+
+1.  At the command prompt, dump the schema post-data from your source database
+    into a `dump_post_data.bak` file, using your source database connection details. Exclude
```

Comment:
In [`pg_dump` documentation](https://www.postgresql.org/docs/current/app-pgdump.html), `.sql` is used for normal dumps and `.dump` is used for `-Fc` format.

```suggestion
    into a `dump_post_data.dump` file, using your source database connection details. Exclude
```

Reply:
Done.